coordinator/src/orderbook/db/orders.rs

```diff
@@ -181,24 +181,12 @@ impl From<OrderbookNewOrder> for NewOrder {
     }
 }
 
-pub fn all(
-    conn: &mut PgConnection,
-    show_expired: bool,
-    show_failed: bool,
```

Comment:
❓ Why did you remove this?

I have no opinion on it, but maybe this was useful to someone? Cc @bonomat.

Reply:
because it was unused.

Reply:
It was part of the HTTP API of the coordinator. In theory someone can query this endpoint to get information about the orderbook.

Reply:
Hmm.. Unless we were using it for some admin task (in which case that api should be in the admin file) it is not used by either the app nor the maker. (Unless I am missing something.

Happy to add it back if we think it's needed. 🙂

Reply:
Note, even if it is used for an admin task, I think it would be better to connect to the database directly for this query. (instead of exposing it publicly)

Reply:
> I think it would be better to connect to the database directly for this query.

That's a good point! Let's remove it then.

Reply:
No hard feelings on this :)